requirements.in

```diff
@@ -1,4 +1,4 @@
-Django>3.2<4
+Django>=3.2.14<4
```

Comment:
Why 3.2.14 specifically? Patch releases should virtually never break anything.

Reply:
Valid point 👍